privacyidea/lib/config.py

```diff
@@ -505,10 +505,9 @@ def get_resolver_class_dict():
             obj = getattr(module, name)
             # There are other classes like HMAC in the lib.tokens module,
             # which we do not want to load.
-            if inspect.isclass(obj) and (issubclass(obj, UserIdResolver) or
-                                             obj == UserIdResolver):
-                # We must not process imported classes!
-                # if obj.__module__ == module.__name__:
```

Comment:
The obj.__module__ == module.__name__ check is terribly important here. Since the kerberos resolver imports and extends the ldap resolver, the loop that populates the resolver list will over ride the kerberos entry with the ldap entry in the hash. This check prevents that and also removes errant UserIDResolver entries (multiple of them) from the computed resolver list.